src/Tickets/Commerce/Gateways/Stripe/Stripe_Elements.php

```diff
@@ -54,8 +53,15 @@ public function include_payment_element() {
 
 		$payment_methods = ( new Merchant() )->get_payment_method_types();
 
-		// Don't load the Payment Element if just the Credit Card method is selected.
-		return ! ( 1 === count( $payment_methods ) && 'card' === $payment_methods[0] );
+		if ( 1 > count( $payment_methods ) ) {
```

Comment:
```suggestion
		if ( 1 < count( $payment_methods ) ) {
```

Shouldn't this be like that instead ?

Reply:
Yes, this is a good catch, thank you!